assets/data/cards_pt-br.json

```
@@ -482,6 +482,72 @@
         "Conceito",
         "Back-end"
       ]
+    },
+    {
+      "title": "NaN",
+      "description": "A propriedade global NaN é um valor especial que significa Not-A-Number(Não é um número).",
+      "tags": [
+        "Conceito",
+        "Back-end",
+        "Front-end"
+      ]
+    },
+    {
+      "title": "Null",
+      "description": "Null é um literal que representa um valor nulo ou vazio.",
```

Acredito que podemos detalhar melhor aqui também. O que acha?
```suggestion
      "description": "Null é uma palavra usada na programação para indicar que algo está vazio ou que não tem nenhum valor. É como se fosse uma caixa vazia que ainda não recebeu nada.",
```

---

Acho que pode inclusive fazer um referência cruzada entre `Null` e `undefined`, citando que podem ser usadas com o mesmo sentido ou não.